wpt/tests/serialization/serialization.py

```diff
@@ -0,0 +1,206 @@
+
+
+@pytest.mark.asyncio
+async def test_serialization_number(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("123", {
+        "type": "number",
+        "value": 123}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("0.56", {
+        "type": "number",
+        "value": 0.56}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_specialNumber(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("+Infinity", {
+        "type": "number",
+        "value": "+Infinity"}, send_blocking_command, context_id,
+                               recursive_compare)
+    await assert_serialization("-Infinity", {
+        "type": "number",
+        "value": "-Infinity"}, send_blocking_command, context_id,
+                               recursive_compare)
+    await assert_serialization("-0", {
+        "type": "number",
+        "value": "-0"}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("NaN", {
+        "type": "number",
+        "value": "NaN"}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_bool(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("true", {
+        "type": "boolean",
+        "value": True}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("false", {
+        "type": "boolean",
+        "value": False}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_function(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("function(){}", {
+        "type": "function",
+        "objectId": "__any_value__"
+    }, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_object(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("{'foo': {'bar': 'baz'}, 'qux': 'quux'}", {
+        "type": "object",
+        "objectId": "__any_value__",
+        "value": [[
+            "foo", {
+                "type": "object",
+                "objectId": "__any_value__"}], [
+            "qux", {
+                "type": "string",
+                "value": "quux"}]]}, send_blocking_command, context_id,
+                               recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_array(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("[1, 'a', {foo: 'bar'}, [2,[3,4]]]", {
+        "type": "array",
+        "objectId": "__any_value__",
+        "value": [{
+            "type": "number",
+            "value": 1
+        }, {
+            "type": "string",
+            "value": "a"
+        }, {
+            "type": "object",
```

Comment:
No `{foo: 'bar'}` here? If testing that here isn't important, then just an empty object in the input would be fine.

Reply:
`max_depth=1`